x-pack/plugins/ml/public/application/trained_models/models_management/models_list.tsx

```diff
@@ -400,9 +452,14 @@ export const ModelsList: FC = () => {
       type: 'icon',
       isPrimary: true,
       available: (item) => item.model_type === 'pytorch',
-      enabled: (item) => !isPopulatedObject(item.pipelines),
+      enabled: (item) =>
+        !isLoading &&
+        !isPopulatedObject(item.pipelines) &&
+        isPopulatedObject(item.stats?.deployment_stats) &&
+        item.stats?.deployment_stats?.state !== DEPLOYMENT_STATE.STOPPING,
```

Comment:
should this be stopping or stopped?
I see a previous commit had it as stopped

Reply:
it was my mistake I just fixed :) 
according to https://www.elastic.co/guide/en/elasticsearch/reference/master/get-trained-model-deployment-stats.html there is only `stopping` state